.github/workflows/setup_env_workflow.yml

```diff
@@ -0,0 +1,49 @@
+name: Setup Envrionment
+
+on:
+  workflow_call:
+    inputs:
+      taskType:
+        required: true
+        type: string
+
+jobs:
+  run_unit_tests:
```

Comment:
Maybe use a different name, such as `webinizer_ci`?

Reply:
do you mean the use `webinizer_ci` as the job name ?

Reply:
Or a different name on your preference. Otherwise there is run_unit_tests job in both webinizer_ci.yml and setup_env_workflow.yml.

Reply:
Exactly, got it, will update it. 
Emm, maybe the workflow's name `setup_env_workflow` is not appropriate either, cause we  execute the `unit_test` and `linting_check` process except setting up environment.

How about changing the workflow name from `setup_env_workflow` to `shared_workflow`, and changing the job name from `run_unit_tests` to `webinizer_ci`

Reply:
> Exactly, got it, will update it. Emm, maybe the workflow's name `setup_env_workflow` is not appropriate either, cause we execute the `unit_test` and `linting_check` process except setting up environment.
> 
> How about changing the workflow name from `setup_env_workflow` to `shared_workflow`, and changing the job name from `run_unit_tests` to `webinizer_ci`

Good idea, please go ahead :)